azure-pipelines.bundlesize.yml

```diff
@@ -27,26 +27,4 @@ jobs:
       - script: yarn bundlesizecollect
```

Comment:
## [🕵 FluentUIV7 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30427&commitId=5bfa70ed932ce28d634469d9aca30c2b093a69ac&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button Action</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Action.Root - RTL.default.chromium.png|0| Added|
|Button Action.Root - RTL.hover.chromium.png|0| Added|
|Button Action.Root - RTL.pressed.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v7"/>